frontend/src/components/common/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -44,14 +44,20 @@ export default class ErrorBoundary extends Component<ErrorBoundaryProps, State>
   render() {
     const { error } = this.state;
     if (error) {
+      console.error('ErrorBoundary:', error);
       store.dispatch(eventAction({ type: HeadlampEventType.ERROR_BOUNDARY, data: error }));
     }
     if (!error) {
       return this.props.children;
     }
+    if (!this.props.fallback) {
```

Comment:
I'm going to do it in a separate PR. VSCode by default uses the bundled typescript version instead of the one from the workspace and in this case there's an error in the latest ts version but not in the 4.5.x. So it compiles fine but in IDE you see an error.